integration-tests/tests/features/@change-email-address.feature

```diff
@@ -22,4 +22,16 @@ Feature: Change email address
     And the page has finished loading
     Then the page meets our accessibility standards
     And the page title is prefixed with "You’ve changed your email address"
-    And I am shown a message confirming that my email address has been changed to "new.email.address@test.com"
\ No newline at end of file
+    And I am shown a message confirming that my email address has been changed to "new.email.address@test.com"
+
+  Scenario: Email address can't be used
+    Given I go to the "Root" page
+    And I sign in as the "default" user
+    Then I go to the "Security" page
+    And I click the "Change email address" link
+    Then I enter and submit my password "Pa55w0rd!"
+    Then I enter and submit my new email address "fail.email.check@test.com"
+    Then I enter and submit the code "123456" sent to my new email address "fail.email.check@test.com"
```

Comment:
I think I'm being really slow here, but what's configuring this test to give us the error page?